winpr/libwinpr/sspi/NTLM/ntlm_compute.c

```diff
@@ -545,7 +653,11 @@ void ntlm_generate_random_session_key(NTLM_CONTEXT* context)
 
 void ntlm_generate_exported_session_key(NTLM_CONTEXT* context)
 {
-	CopyMemory(context->ExportedSessionKey, context->RandomSessionKey, 16);
+	WINPR_ASSERT(context);
+	WINPR_ASSERT(sizeof(context->ExportedSessionKey) == sizeof(context->RandomSessionKey));
```

Comment:
I don't see any reason for this assert, this is the design `NTLM_CONTEXT`, we're not going to check at every call the design of this struct !